src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+
 public class FileWork {
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] readFromFile;
```

Comment:
move this declaration closer to its first usage
and use more informative naming